subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -238,48 +266,48 @@ static void olcp_ind_handler(struct bt_conn *conn,
 
 	BT_DBG("OLCP indication");
 
-	if (op_code == BT_OTS_OLCP_PROC_RESP) {
-		enum bt_ots_olcp_proc_type req_opcode =
+	if (op_code == BT_GATT_OTS_OLCP_PROC_RESP) {
+		enum bt_gatt_ots_olcp_proc_type req_opcode =
 			net_buf_simple_pull_u8(&net_buf);
-		enum bt_ots_olcp_res_code result_code =
+		enum bt_gatt_ots_olcp_res_code result_code =
 			net_buf_simple_pull_u8(&net_buf);
 
 		print_olcp_response(req_opcode, result_code);
 
 		switch (req_opcode) {
-		case BT_OTS_OLCP_PROC_FIRST:
+		case BT_GATT_OTS_OLCP_PROC_FIRST:
 			BT_DBG("First");
 			on_object_selected(conn, result_code, otc_inst);
 			break;
-		case BT_OTS_OLCP_PROC_LAST:
+		case BT_GATT_OTS_OLCP_PROC_LAST:
 			BT_DBG("Last");
 			on_object_selected(conn, result_code, otc_inst);
 			break;
-		case BT_OTS_OLCP_PROC_PREV:
+		case BT_GATT_OTS_OLCP_PROC_PREV:
 			BT_DBG("Previous");
 			on_object_selected(conn, result_code, otc_inst);
 			break;
-		case BT_OTS_OLCP_PROC_NEXT:
+		case BT_GATT_OTS_OLCP_PROC_NEXT:
 			BT_DBG("Next");
 			on_object_selected(conn, result_code, otc_inst);
 			break;
-		case BT_OTS_OLCP_PROC_GOTO:
+		case BT_GATT_OTS_OLCP_PROC_GOTO:
 			BT_DBG("Goto");
 			on_object_selected(conn, result_code, otc_inst);
```

Comment:
I believe with the GOTO procedure we know the object ID and we shouldn't reset it to an unknown ID. But again this can probably be fixed in a follow up PR.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/42782